policies/templates/restrict-host-ingress.yaml

```
@@ -0,0 +1,32 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: {{ include "policies.fullname" . }}-check-for-host
```

```suggestion
  name: {{ include "policies.fullname" . }}-restrict-host-ingress
```

Please, ensure that the policy name matches that of the file. Same for all others.